dialogue/TownNPCs/ShovelGuy/ShovelGuy2.json

```diff
@@ -21,7 +21,7 @@
     ],
     "pl": [
       "Już pracowałem nad tym przez jakiś czas, i nie wygląda na to, że skończę w najbliższym czasie.",
-      "Mogę widzieć wiele ludzi mijając mnie, ale ciągle tu kopię przez tyle dni.",
+      "Mogę widzieć wiele ludzi mijających mnie, ale ciągle tu kopię przez tyle dni.",
       "Ale jedna rzecz, której nie mogę ogarnąć...",
       "Dlaczego ludzie nazywają mnie \"Łopatowiec\"?",
       "Dopiero zacząłem kopać kilka dni temu!"
```

Comment:
I'd move dopiero to be after "kopać" rather than at the beggining of the sentence

Reply:
Intentionally placed at the beginning for emphasis.